samples/cpp/speech_sample/speech_sample.hpp

```diff
@@ -140,6 +145,9 @@ DEFINE_string(exec_target, "", execution_target_message);
 /// \brief GNA compile target <br>
 DEFINE_string(compile_target, "", compile_target_message);
 
+/// \brief GNA log level <br>
+DEFINE_bool(log, false, enable_log_message);
```

Comment:
Maybe define it as string? In order to extend usage of this variable in future for other values (`INFO`, `DEBUG` and so on)?

Reply:
I was thinking about this, but went for the minimum as a first step. Will do it.